app/src/main/java/protect/card_locker/preferences/myAppCompatActivity.java

```diff
@@ -0,0 +1,58 @@
+package protect.card_locker.preferences;
+
+import android.content.SharedPreferences;
+import android.content.res.Resources;
+import android.widget.Toast;
+
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.preference.PreferenceManager;
+
+import protect.card_locker.R;
+
+public class myAppCompatActivity extends AppCompatActivity {
+
+    SharedPreferences pref;
+//
+        @Override
+    public Resources.Theme getTheme() {
+        Resources.Theme theme = super.getTheme();
+            pref = PreferenceManager
+                    .getDefaultSharedPreferences(getApplicationContext());
+            String themeName = pref.getString("pref_theme_color", "App theme");
+            switch (themeName){
+                case "Pink theme":
+                    theme.applyStyle(R.style.AppTheme_pink, true);
+                    break;
+                case "Red theme":
```

Comment:
The red theme looks very close to the Catima theme (because Catima is already a type of red), can we remove it because it is too similar?

Reply:
yes sure